docs/content/guides/dagster/how-assets-relate-to-ops-and-graphs.mdx

```diff
@@ -0,0 +1,156 @@
+---
+title: Understanding how software-defined assets relate to ops and graphs | Dagster Docs
+description: Dagster’s main abstraction for building data pipelines is the software-defined asset. However, Dagster also has ops and graphs. How do these relate? When should you use one over the other?
+---
+
+# Understanding how software-defined assets relate to ops and graphs
+
+Dagster’s main abstraction for building data pipelines is the _software-defined asset_. However, Dagster also has abstractions called _ops_ and _graphs_.
+
+If you're not sure which one to use, this guide is for you. In this guide, we'll cover:
+
+- When you should assets or ops and graphs
+- How assets, ops, and graphs relate to each other
+- Why Dagster offers all three abstractions
```

Comment:
@sryza I redid the intro a little bit to make it more skimmable, but it also made me think we don't touch on the last item in this list: `Why does Dagster offer assets and ops and graphs?`

Is this something worth covering in this iteration? If not, we can take it out for now.

Reply:
I think we can take this out for now